### ISSUE TITLE
placeholder: support aws-java-sdk-events functionality

### ISSUE THREAD

**Issue by glebsts** [classmethod/gradle-aws-plugin#72]
My usecase is that I deploy lambda code with this plugin, and I would like to add trigger for invoking this lambda every 5 min. Looked into [AWS docs ](http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/RunLambdaSchedule.html)- there are events and rules, but currently I have to write cloudformation jsons for that.

### Acceptance criteria ###
* at least create, delete, enable and disable rule functionality is supported
* stretch goal: invoking test event is supported (to test lambda invocation after deploy)

**Comment by DALDEI** [comment]
Test lamba invocation is already supported using the invokeFunction task.  I use it in my builds/tests regularly.  Partituclarly  useful is the ability to get the log events from the invocation.

Triggers are an AWS Console simplification of a small set of resource specific integrations to lamba.
I suggest that that core event support be implemented there - on per resource/api basis.
For example S3 event triggers on S3.

"Events" and "Rules" are a CloudWatch feature.  I suggest that they be implemented as part of a CloudWatch plugin -- which would be extremly useful.   The CloudWatch API is difficult to get right,
and having a reliably method of configuring events, filters, logs, and custom metrics is very useful.

IN addition, the ability to extract/search for CloudWatch logs (given a filter), and to extract raw CloudWatch Metric data (given metric dimensions, timerange) would be wonderful.

Bonus points -- integration into a graphing plugin or standard format like "dot"

**Comment by glebsts** [comment]
> Test lamba invocation is already supported using the invokeFunction task
Yes, I said "invoking test event is supported" and then just explained my use case. I do invoke lambdas now, but I want to invoke event (that invokes lambda, but it doesn't matter).
You just made scope too big for me mb :)
